plugins/discovery/discovery.go

```diff
@@ -87,8 +89,31 @@ func New(manager *plugins.Manager, opts ...func(*Discovery)) (*Discovery, error)
 		return result, nil
 	}
 
-	if names := manager.Config.PluginNames(); len(names) > 0 {
-		return nil, fmt.Errorf("discovery prohibits manual configuration of %v", strings.Join(names, " and "))
+	result.localConficOverrides = make(map[string]json.RawMessage)
+	for _, name := range manager.Config.PluginNames() {
+		factory := result.factories[name]
+		var pluginConfig json.RawMessage
+		if name == "decision_logs" {
+			if err := logs.AllowConfigOverride(manager.Config.DecisionLogs); err != nil {
```

Comment:
🤔 so decision logs doesn't live in `manager.Config.Plugins["decision_logs"]`? Unfortunate 😳

Reply:
yeah, that is actually how I started 😄 . All the "initial" plugins also do not implement the `Factory` interface so I guess this is all structures from the early days. It also leads to quite a bit of boilerplate further down...